packages/components/src/components/page-error/page-error.jsx

```diff
@@ -4,56 +4,80 @@ import React from 'react';
 import ButtonLink from '../button-link/button-link.jsx';
 import DesktopWrapper from '../desktop-wrapper/desktop-wrapper.jsx';
 
-const PageError = ({ buttonOnClick, classNameImage, header, image_url, messages, redirect_label, redirect_url }) => (
-    // if image_url is passed we should split the page to two columns and left-align messages
-    <div className={classNames('dc-page-error__container', { 'dc-page-error__container--left': !!image_url })}>
-        {!!image_url && (
-            <DesktopWrapper>
-                <img
-                    className={classNameImage}
-                    src={image_url}
-                    alt={'404'}
-                    loading='lazy'
-                    width='771px' // width and height should be specified so it doesn't jump to the right after image loads
-                    height='448px'
-                />
-            </DesktopWrapper>
-        )}
-        <div
-            className={classNames('dc-page-error__box', {
-                'dc-page-error__box--left': !!image_url,
-            })}
-        >
-            <h3 className='dc-page-error__header'>{header}</h3>
+const PageError = ({
+    buttonOnClick,
+    classNameImage,
+    header,
+    image_url,
+    messages,
+    redirect_label,
+    redirect_url,
+    should_clear_error_on_click,
+    setError,
+}) => {
+    const onClickHandler = () => {
+        if (should_clear_error_on_click) {
+            setError(false, null);
+        } else {
+            buttonOnClick();
+        }
+    };
+
+    return (
+        // if image_url is passed we should split the page to two columns and left-align messages
+        <div className={classNames('dc-page-error__container', { 'dc-page-error__container--left': !!image_url })}>
+            {!!image_url && (
+                <DesktopWrapper>
+                    <img
+                        className={classNameImage}
+                        src={image_url}
+                        alt={'404'}
+                        loading='lazy'
+                        width='771px' // width and height should be specified so it doesn't jump to the right after image loads
```

Comment:
how about responsive?, can it be `100vw` instead?

Reply:
I'm not sure if this is in the scope of the card or not. Also I think `100vw` will be inconsistent with the specified height. Let's back to this in case QA raises an issue.